packages/provider/src/utils.ts

```diff
@@ -23,6 +23,46 @@ export const prefixEIP191Message = (message: BytesLike): Uint8Array => {
   }
 }
 
+export const removeEIP191Prefix = (prefixedMessage: Uint8Array): Uint8Array => {
```

Comment:
lets rename this to `trimEIP191Prefix` .. minor thing, but I think "trim" is more common word, which reminds me of like "trimming whitespace"

Reply:
I've updated the PR :)